docs/sources/index.md

```diff
@@ -104,4 +104,12 @@ aliases = ["v1.1", "guides/reference/admin"]
       <img src="/img/docs/logos/icon_cloudwatch.svg">
       <h5>Cloudwatch</h5>
     </a>
+    <a href="{{< relref "features/datasources/mssql.md" >}}" class="nav-cards__item nav-cards__item--ds">
```

Comment:
First, suggest to change the order:
1. Graphite
2. Elasticsearch
3. InfluxDB
4. Prometheus
5. Google Stackdriver
6. Cloudwatch
7. MySQL
8. Postgres
9. Microsoft SQL Server
10. OpenTSDB

Second, the CSS change of nav-cards and nav-cards__item doesn't work that good on mobile/tablet breakpoints for all cards on that page.

Reply:
On second though, how about having 5 items per row instead?